h2o-py/h2o/model/models/anomaly_detection.py

```diff
@@ -10,16 +10,19 @@ class H2OAnomalyDetectionModel(ModelBase):
     def varsplits(self, use_pandas=False):
         """
         Retrieve per-variable split information for a given Isolation Forest model. Output will include:
-        - count - The number of times a variable was used to make a split.
-        - aggregated_split_ratios - The split ratio is defined as "abs(#left_observations - #right_observations) / #before_split".
-                                    Even splits (#left_observations approx the same as #right_observations) contribute
-                                    less to the total aggregated split ratio value for the given feature;
-                                    highly imbalanced splits (eg. #left_observations >> #right_observations) contribute more.
-        - aggregated_split_depths - The sum of all depths of a variable used to make a split. (If a variable is used
-                                    on level N of a tree, then it contributes with N to the total aggregate.)
+        
+        - count
+            The number of times a variable was used to make a split.
+        - aggregated_split_ratios
+            The split ratio is defined as ``abs(#left_observations - #right_observations) / #before_split``.
+            Even splits (``#left_observations`` approx the same as ``#right_observations``) contribute
+            less to the total aggregated split ratio value for the given feature;
+            highly imbalanced splits (eg. ``#left_observations >> #right_observations``) contribute more.
+        - aggregated_split_depths
+            The sum of all depths of a variable used to make a split. 
+            (If a variable is used on level N of a tree, then it contributes with N to the total aggregate.)
```

Comment:
@hannah-tillman looks like the best way to have lists with long descriptions for each item